docs/content/learn/tutorials/component/_index.md

```diff
@@ -0,0 +1,335 @@
+
+{{< tabs XML >}}
+{{% tab %}}
+{{< highlight xml "linenos=table" >}}
+<component-definition xmlns="http://csrc.nist.gov/ns/oscal/1.0"
+    uuid="a7ba800c-a432-44cd-9075-0862cd66da6b">
+  ...  
+  <component uuid="91f646c5-b1b6-4786-9ec3-2305a044e217" type="software">
+      ...
+      <control-implementation
+        uuid="49f0b690-ed9f-4f32-aae0-625b77aa6d27"
+        source="../../../content/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_MODERATE-baseline_profile.xml">
+        <description>
+          <p>MongoDB control implementations for NIST SP 800-53rev5.</p>
+        </description>
+        <implemented-requirement
+          uuid="cf8338c5-fb6e-4593-a4a8-b3c4946ee2a0"
+          control-id="sc-8.1">
+          <description>
+              <p>MongoDB supports TLS 1.x to encrypt data in transit, preventing unauthorized disclosure or changes to information during transmission. To implement TLS, set the PEMKeyFile option in the configuration file /etc/mongod.conf to the certificate file's path and restart the the component.</p>
+          </description>
+        </implemented-requirement>
+      </control-implementation>
+  </component>
+  ...
+</component-definition>
+{{< /highlight >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+In the example above, we demonstrated how to use OSCAL to document the MongoDB software component's control implementation for NIST control SC-8(1). However, one could document how the same component also satisfies controls from other security risk management frameworks (e.g., FedRAMP, ISO 27002, etc.) by simply adding additional `<control-implementation>` elements for each.  Likewise, for each `<control-implementation>`, we can add `<implemented-requirement>` elements for every control that can be implemented by the component.  
+
+## The Final Component Definition
+
+Combining all the content described in this tutorial, we can produce this final component definition model.  Having these implementation details readily available in an OSCAL component definition means they can be referenced in an OSCAL SSP and should give SSP authors helpful content for their control implementation statements.
```

Comment:
the statement reads: "means they can be *referenced* in an OSCAL SSP". I think the information is used to populate the SSP not to just reference it.